management-api-agent-4.x/src/main/java/com/datastax/mgmtapi/shim/CassandraAPI4x.java

```diff
@@ -67,6 +68,29 @@
 
     private static final Supplier<SeedProvider> seedProvider = Suppliers.memoize(() -> new K8SeedProvider4x());
 
+    public void enableFullQuerylog() 
+    {
+        FullQueryLoggerOptions fqlOpts = DatabaseDescriptor.getFullQueryLogOptions();
```

Comment:
When I briefly looked at this part, I wasn't sure if you had to give the service all the options or not. Do you know if there is a simple `enable` function that will just use whatever FQL config Cassandra has in its `cassandra.yaml`? I so, this can be simplified as there is no need to grab the config it already has and pass to the service call if the service will just use the same config be default. I think making this call here would be more for if the Management API REST endpoint allowed for configuration overrides, which I don't think we want to do at this point.

Reply:
I think I do. I don't believe there is a method to enable FQL that doesn't take the parms (happy to be wrong though!)